llm/main.py

```diff
@@ -71,3 +77,12 @@ def ready():
 def httpsflaskexample(req: https_fn.Request) -> https_fn.Response:
     with app.request_context(req.environ):
         return app.full_dispatch_request()
+
+
+@on_document_created(
+    secrets=["OPENAI_DEV", "OPENAI_PROD"],
+    document="generalCourts/{session_id}/bills/{bill_id}",
+)
+def add_summary_on_document_created(event: Event[DocumentSnapshot | None]) -> None:
+    set_openai_api_key()
+    bill_on_document_created.run_trigger(event)
```

Comment:
I'm not sure if we have a convention here in typescript-land, but I would prefer to keep the implementations of the triggers separate from their trigger definition.